js/index.js

```diff
@@ -29,7 +29,7 @@ function updateUrl(state, defaultState) {
             diff[key] = state[key].toString();
         }
     }
-    window.history.replaceState(null, "", "/?" + new URLSearchParams(diff).toString());
+    window.history.replaceState(null, "", window.location.pathname + "?" + new URLSearchParams(diff).toString());
```

Comment:
Wouldn't be enough to just remove the slash `/` in here? Is the `window.location.pathname` really necessary?